dataland-keycloak/dataland_theme/login/templates/register.ftl

```diff
@@ -64,6 +95,22 @@
                 LOGIN TO ACCOUNT
             </Button>
         </div>
+
+        <script>
+            let submit_button = document.getElementById("register_button");
+
+            let first_name_field = document.getElementById("firstName");
+            let last_name_field = document.getElementById("lastName");
+
+            submit_button.addEventListener("click", function () {
+                console.log("changing names");
+                if(!first_name_field.value) first_name_field.value = "-";
+                if(!last_name_field.value) last_name_field.value = "-";
+                console.log("first name: " + first_name_field.value);
```

Comment:
please remover logger if unnecessary

Reply:
in my version of the code, the logger doesnt exist (anymore) but even before addressing your comment

Reply:
Is no longer there